app/api/__init__.py

```diff
@@ -121,32 +121,33 @@
 # social_links
 api.route(SocialLinkList, 'social_link_list', '/events/<int:id>/social-links', '/events/<identifier>/social-links')
 api.route(SocialLinkDetail, 'social_link_detail',
-          '/social-links/<int:id>', '/events/<int:event_id>/social-links')
+          '/social-links/<int:id>', '/events/<int:event_id>/social-links',
```

Comment:
Why is `/events/<int:event_id>/social-links` and `/events/<event_identifier>/social-links` here ? This is socialLinkDetail